verifier/credentialsConfig.go

```diff
@@ -68,24 +62,47 @@ func InitServiceBackedCredentialsConfig(repoConfig *config.ConfigRepo) (credenti
 func (cc ServiceBackedCredentialsConfig) fillStaticValues() {
 	for _, configuredService := range cc.initialConfig.Services {
 		logging.Log().Debugf("Add to service cache: %s", configuredService.Id)
-		cc.serviceCache.Add(configuredService.Id, configuredService, cache.NoExpiration)
+		err := common.GlobalCache.ServiceCache.Add(configuredService.Id, configuredService, cache.NoExpiration)
+		if err != nil {
+			logging.Log().Errorf("failed caching configured service in fillStaticValues(): %v", err)
+		}
 	}
 }
 
-func (cc ServiceBackedCredentialsConfig) fillCache(ctx context.Context) {
-	client := *(cc.configClient)
-	services, err := client.GetServices()
+func (cc ServiceBackedCredentialsConfig) fillCache(context.Context) {
+	configClient := *(cc.configClient)
+	services, err := configClient.GetServices()
 	if err != nil {
 		logging.Log().Warnf("Was not able to update the credentials config from the external service. Will try again. Err: %v.", err)
 		return
 	}
 	for _, configuredService := range services {
-		cc.serviceCache.Add(configuredService.Id, configuredService, cache.NoExpiration)
+		err := common.GlobalCache.ServiceCache.Add(configuredService.Id, configuredService, cache.NoExpiration)
```

Comment:
What if a service gets removed from the CCS?

Reply:
You're right, expiration should be changed to 'DEFAULT', and then it'll expire after given expiration time